mock-schema.json

```diff
@@ -5,14 +5,18 @@
   },
   "groups": [
     {
-      "measures": [
+      "metrics": [
```

Comment:
Why metrics, what is wrong with measures? We want to get rid of this dual naming

Reply:
Because mock-js does not work with "measure" metadata object

Reply:
I mean as long as we will call "metrics" on our server "metrics", mock-js will use that name to keep the consistency.

And by the way... "measure" means "fact" and "calculated measure" means "metric" :-)

Reply:
I got it, thx for explanation